apps/aehttp/test/aehttp_ws_test_utils.erl

```diff
@@ -59,21 +59,28 @@ start_link_channel(Host, Port, RoleA, Opts) when is_atom(RoleA) ->
     Role = atom_to_binary(RoleA, utf8),
     WsAddress = make_channel_connect_address(Host, Port, Role, Opts),
     ct:log("connecting to Channel ~p as ~p", [WsAddress, Role]),
-    {ok, Pid} = websocket_client:start_link(WsAddress, ?MODULE, self()),
+    {ok, Pid} = websocket_client:start_link(WsAddress, ?MODULE, self(),
+                                            extra_headers()),
     wait_for_connect(Pid).
 
 start_channel(Host, Port, RoleA, Opts) when is_atom(RoleA) ->
     Role = atom_to_binary(RoleA, utf8),
     WsAddress = make_channel_connect_address(Host, Port, Role, Opts),
     ct:log("connecting to Channel ~s as ~p", [iolist_to_binary(WsAddress), Role]),
-    {ok, Pid} = websocket_client:start(WsAddress, ?MODULE, self()),
+    %% There is no websocket_client:start/4 ...
+    {ok, Pid} = websocket_client:start_link(WsAddress, ?MODULE, self(),
+                                            extra_headers()),
+    unlink(Pid),
     case wait_for_connect(Pid) of
         {ok, Pid} = Res ->
             set_role(Pid, RoleA),
             Res;
         {error, _} = Err -> Err
     end.
 
+extra_headers() ->
+    [{extra_headers, [{<<"Content-Type">>, <<"application/jsonrpc">>}]}].
```

Comment:
Isn't that `application/json-rpc`?

Reply:
Not sure where I got that from. [Looking it up](https://www.jsonrpc.org/historical/json-rpc-over-http.html):
> Content-Type SHOULD be 'application/json-rpc' but MAY be 'application/json' or 'application/jsonrequest'

Reply:
> Not sure where I got that from. [Looking it up](https://www.jsonrpc.org/historical/json-rpc-over-http.html):

I read in a section of the code of this PR `<<"jsonrpc">> => <<"2.0">>` so is the spec followed the 2.0 one at https://www.jsonrpc.org/specification ? I see no content type / media type there. And, more specifically, I see no [media type registered with IANA](https://www.iana.org/assignments/media-types/media-types.xhtml).

Reply:
@uwiger Yes, I saw that too, but still `-` is missing, and these specs are for JSON RPC 1.2 ? 🤔